config/environment.rb

```diff
@@ -1,3 +1,4 @@
+Rails.env = 'development'
```

Comment:
Why is this set like this?

Reply:
If I remember correctly, I was trying to set something up that wasn't working properly and that was something that came up. I don't think it is necessary.

Reply:
okay then it should get removed